case_studies/windpark_optimization_nordex.md

```diff
@@ -0,0 +1,81 @@
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/10322094/121471521-1f3e1c00-c9c0-11eb-901a-0d44f8aba254.png" alt="layout_1" width="500"/>
+</p>
+
+## Network Generation and Enrichment
+
+Contour lines as such represent a constant value, in our case the elevation. The resulting curve joins points of equal value. If one were to connect points (vertices) of neighboring curves which each other eventually a topology can be derived yielding a [graph](https://en.wikipedia.org/wiki/Graph_(discrete_mathematics) "Graph Theory") consisting of vertices and edges. We can hijack [Delaunay's approach for triangulating](https://en.wikipedia.org/wiki/Delaunay_triangulation "Delaunay Triangulation") a set of points to achieve this efficiently which will produce a set of non-intersecting triangles connecting neighboring contour lines with each other. As every point features an elevation value we can go one step further and assign a cost for this connection (edge) by computing the elevation gain given the length of the connection (run) and the vertical difference (rise).
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/10322094/121474513-7940e080-c9c4-11eb-8c32-2261096f59fe.png" alt="countours" width="500"/>
+</p>
+
+At this point this graph merely knows about the elevation, i.e. shortest path algorithms will naturally avoid costly edges (steep!), so it's actually what we wanted to achieve. The reason behind is that large trucks loaded with heavy turbines will have a hard time to drive on them.
+On the other hand the terrain may enclose natural habitats, waterbodies or regions which additionally should be considered in the path finding between points on the map. While a lake will be a hard restriction, a rocky area may rather be penalised as it will be hard to build paved roads on. Any of these features can be described as areas which as overlays can spatially effect the base network by intersection causing a hard removal of the edge or an update of the edge cost.
+
+
+<p align="center">
+<img src="https://user-images.githubusercontent.com/10322094/121502297-515f7600-c9e0-11eb-8eca-cb6b4ccfc007.png" alt="countours" width="500"/>
+</p>
+
+
+The last step in the preparation of the graph is to bake roads, such as jeep tracks, which already exist into the topology. These are carefully selected by the wind farm planners and as no actual building cost is involved, the cost can be neglected for this subset of edges.
```

Comment:
```suggestion
The last step in the preparation of the graph is to integrate existing roads, such as jeep tracks, into the graph's topology. These are carefully selected by the wind farm planners and as no actual building cost is involved, the edge cost can be set to a minimal value, so that the path finding will prefer already existing roads.
```